crates/sui-graphql-rpc/schema.graphql

```diff
@@ -0,0 +1,840 @@
+# Copyright (c) Mysten Labs, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+# Open Questions
+# --------------
+# - Rules for nullabilit/non-nullability
+# - Pattern for multi-gets
+# - Encoding RPC errors
+# - `type` (vs `scalar`) for Move Types
```

Comment:
Proposal uses a "rich" (`type`) representation of Move Types that contains a field with a flattened/scalar representation.